ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__periodics.yaml

```diff
@@ -172,7 +172,6 @@ tests:
     env:
       HYPERSHIFT_NODE_COUNT: "2"
       LVM_OPERATOR_SUB_CHANNEL: stable-4.14
-      MCE_INDEX_IMAGE: brew.registry.redhat.io/rh-osbs/iib:716901
```

Comment:
we also have a reference for it in `ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-ref.yaml` . Can we drop it from there as well for consistency?

Reply:
hypershift-mce-install-disconnected will be deprecated